environment.devenv.yml

```diff
@@ -17,7 +17,14 @@ dependencies:
     - cmake >=3.5.2
 
     {% if os.sys.platform != 'win32' %}
-    - clangdev>=6.0.0
+    - conda-forge::gcc_linux-64==7.3.0
+    - conda-forge::gcc_impl_linux-64==7.3.0
+    - conda-forge::gxx_linux-64==7.3.0
+    - conda-forge::gxx_impl_linux-64==7.3.0
```

Comment:
These packages cannot be found at the moment. Further investigation is required.